js/plugins/conviva.js

```diff
@@ -500,6 +512,7 @@ var ConvivaAnalyticsPlugin = function(framework)
    */
   var resetContentState = function()
   {
+    streamUrl = null;
```

Comment:
currentConvivaSessionKey is set to the NO_SESSION_KEY value in the clearLastSession function. streamType we will need to double check how often it updates.